javascript/javascript_in_the_real_world/what_is_es6.md

```diff
@@ -9,7 +9,7 @@ You have probably also come across articles talking about features in ES7 or ES8
 - [This article](https://codeburst.io/javascript-wtf-is-es6-es8-es-2017-ecmascript-dca859e4821c) provides a nice clean explanation and timeline of the various ECMAScript releases.
 - [This document](https://github.com/lukehoban/es6features) outlines all the new features that showed up in ES6. As we've mentioned you've already been using many of these, though there are a few we haven't specifically covered yet.
 
-The _problem_ with JavaScript constantly updating and adding features is that it sometimes takes web-browsers a while to catch up and implement new features once they've been released. At the current time all modern browsers (Chrome, Firefox, Safari and Edge) support _all_ of ES6, and _most_ of ES7, but older browsers (various versions of Internet Explorer for instance) do not. This means, unfortunately, that if you write code that uses these new features it __will not run__ in browsers that do not support it.
+The _problem_ with JavaScript constantly updating and adding features is that it sometimes takes web-browsers a while to catch up and implement new features once they've been released. At the current time all modern browsers (Chrome, Firefox, Safari and Edge) support _all_ of ES6, and _most_ of ES7, but older browsers, such as Internet Explorer (recently disabled) and its various versions did not. This means, unfortunately, that if you write code that uses these new features it __will not run__ in browsers that do not support it.
```

Comment:
I'm not entirely sure how important it is to mention this here. From what I understand IE may be disabled on Windows 10/11 at a certain update, but IE can still be used prior to that update/older versions of Windows. The main takeaway from this paragraph is just that JavaScript updates may not be available in all browsers immediately or at all.